docs/LOCAL_SETUP.md

```diff
@@ -201,6 +203,15 @@ This will launch a dedicated server on port 3045 (default setting) and allow inc
 
 This will launch the CodeHarbor web application server on port 7500 (default setting) and allow incoming connections from your browser.
 
+3. Solid Queue supervisor (optional):
+
+  ```shell
+  bundle exec rake solid_queue:start
+  ```
+  
+Solid Queue is currently only used to synchronize task metadata with the "NBP Datenraum".
+The synchronization is disabled by default. Unless you enable it, you can skip starting the supervisor.
```

Comment:
Are we sure some other Rails components are not using the default job backend, e.g., for sending e-mails or purging deleted files (test on the profile page with the avatar)?

Reply:
Thanks!